Source/Particles/Collision/BackgroundMCCCollision.cpp

```diff
@@ -118,7 +118,8 @@ BackgroundMCCCollision::get_nu_max(amrex::Vector<MCCProcess> const& mcc_processe
     using namespace amrex::literals;
     amrex::Real nu, nu_max = 0.0;
 
-    for (double E = 1e-4; E < 5000; E+=0.2) {
+    /* NOTE: Was E make a double intentionally? */
+    for (amrex::Real E = 1e-4_rt; E < 5000._rt; E+=0.2_rt) {
```

Comment:
@roelof-groenewald Is this change to use amrex::Real Ok?

Reply:
Yeap that change is fine. Seeing as the MCC unit tests pass the extra precision is clearly not crucial. Thanks for checking!